.github/workflows/release.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     tags:
       - 'v*'
+    branches:
+      - '**'
```

Comment:
Was this just for testing?  Is it intentional to leave this in?

Reply:
It's intentional.  It allows the code to be built in any branch for CI testing purposes but not released unless it's the main branch with a version tag.

Reply:
For example: https://github.com/kionsoftware/terraform-provider-kion/actions/runs/9118768981